core/ecschema-metadata/src/Deserialization/JsonProps.ts

```diff
@@ -34,6 +34,16 @@ export interface SchemaProps {
   readonly customAttributes?: Array<{ [value: string]: any }>;
 }
 
+/**
+ * @beta
+ */
+export interface SchemaKeyProps {

```

Comment:
This props interface should be documented ... point back to SchemaKey because the main docs should be there

Reply:
Nitpicky but please follow conventions by putting release tag last, and prefer inline links.
```
/** JSON Object interface used to deserialize into a [[SchemaKey]].
 * @beta
 */
```